packages/phone-number-privacy/monitor/src/test.ts

```diff
@@ -35,7 +38,27 @@ export async function testPNPQuery(
   }
 }
 
-export async function testDomainsQuery(contextName: OdisContextName) {
+export async function testPNPQuotaQuery(
+  blockchainProvider: string,
+  contextName: OdisContextName,
+  timeoutMs?: number
+) {
+  logger.info(`Performing test PNP query for ${CombinerEndpointPNP.PNP_QUOTA}`)
+  try {
+    const odisResponse: PnpClientQuotaStatus = await queryOdisForQuota(
+      blockchainProvider,
+      contextName,
+      timeoutMs
+    )
+    logger.info({ odisResponse }, 'ODIS quota request successful. System is healthy.')
+  } catch (err) {
+    logger.error('ODIS quota request failed.')
+    logger.error({ err })
+    throw err
```

Comment:
why rethrow?

Reply:
just so that the monitor cloud function fails